Tests/AutoKeyboardNavigatorTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AutoKeyboardNavigatorTests.swift
+//  KeyboardSupport-iOSTests
+//
+//  Created by John Davis on 12/18/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import XCTest
+@testable import KeyboardSupport
+
+class AutoKeyboardNavigatorTests: XCTestCase {
```

Comment:
For consistency, we may want to name this file/test class `KeyboardAutoNavigatorTests` instead of `AutoKeyboardNavigatorTests`.

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/b30ef2048cc3b6625c9a1e0dcbe9ecc4cbdbde47